core/lib/generators/solidus/install/app_templates/payment_method/paypal.rb

```diff
@@ -0,0 +1,6 @@
+unless Bundler.locked_gems.dependencies['solidus_frontend']
+  say_status :warning, "Support for frontends other than `solidus_frontend` by `solidus_paypal_commerce_platform` is still in progress.", :yellow
+end
+
+bundle_command 'add solidus_paypal_commerce_platform'
```

Comment:
We should skip this line if the gem is already bundled in the same vein that you're considering on `detect_payment_method_to_install` below.